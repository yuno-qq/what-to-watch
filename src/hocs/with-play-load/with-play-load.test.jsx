import React from 'react'
import {describe, test, expect} from "@jest/globals"
import renderer from 'react-test-renderer'

import withPlayLoad from "./with-play-load.jsx"


describe(`<withPlayLoad> snapshots:`, () => {
  test(`should render component`, () => {
    const MockComponentWrapped = withPlayLoad(() => <div className="MOCK_COMPONENT"></div>)

    const tree = renderer.create(<MockComponentWrapped setActiveItem={() => {}} />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
