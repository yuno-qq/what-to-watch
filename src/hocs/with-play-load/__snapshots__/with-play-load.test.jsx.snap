// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<withPlayLoad> snapshots: should render component 1`] = `
<div
  className="MOCK_COMPONENT"
/>
`;
